t/t0034-broker-getenv.t

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+test_description='Test broker.getenv rpc'
```

Comment:
should add some tests that ensure only instance owner get get things and non-instance owner cant?

Reply:
OK tacked on that extra test.  Thanks - I'll set MWP.